arch/arm/src/armv7-m/arm_vectors.c

```diff
@@ -51,6 +51,47 @@
 #  error ARMV7M_PERIPHERAL_INTERRUPTS must be defined to the number of I/O interrupts to be supported
 #endif
 
+#define ARMV7M_VECTAB_SIZE (ARMV7M_PERIPHERAL_INTERRUPTS + 16)
```

Comment:
why dup the code nuttx/arch/arm/src/armv7-m/arm_ramvec_initialize.c?

Reply:
Remove the code and include "ram_vectors.h", Is it OK ?

Reply:
could be.

Reply:
@xiaoxiang781216 
Thank you for your advice, I have done it.